### DISCUSSION TITLE
用你的Notion笔记本快速搭建一个博客

### DISCUSSION THREAD

**Discussion by tangly1024** [tangly1024/NotionNext#132] [General]
http://mylocalstuff.tangly1024.com:3000/article/notion-next#52f38f9723e8437bb82e028f1cd4be94 

 如果你在使用Notion这款神级笔记本的话，不妨来试试顺手建个网站🤣🤣🤣，这是一款基于NotionAPI的博客系统。源码地址 https://github.com/tangly1024/NotionNext

**Comment by tangly1024** [comment]
测试gittalk

**Comment by JobsQzhang2018** [comment]
学习了

**Comment by SenLief** [comment]
我咋能切换到Next主题呢

**Comment by tangly1024** [comment]
> @SenLief
> 我咋能切换到Next主题呢

先更新一下代码，然后参考我的[blog.config.js](https://github.com/tangly1024/NotionNext/blob/main/blog.config.js)文件重新配置一下😀

**Comment by lengpingyi** [comment]
你好，请问底部的备案信息在哪里更改呢？

**Comment by SenLief** [comment]
> 你好，请问底部的备案信息在哪里更改呢？

Footer.js

**Comment by lengpingyi** [comment]
> @SenLief
> > 你好，请问底部的备案信息在哪里更改呢？
> 
> Footer.js

非常感谢！

**Comment by SenLief** [comment]
对了，之前那个三栏的主题没有了吗？

**Comment by tangly1024** [comment]
> 对了，之前那个三栏的主题没有了吗？

@SenLief 

之前三栏的主题，我正在重构开发中，目前还未完善，现在只是一个初步版本 

切换主题说明
[主题切换](https://docs.tangly1024.com/zh/advanced/themes)

预览HEXO主题地址
[Hexo主题](https://notionnext-8kk37uaz7-tlyong1992.vercel.app/)
